tests/testthat/test-create_sample_histograms.R

```diff
@@ -0,0 +1,25 @@
+# generate helper data
+pop <- generate_virtual_pop(N=100, var_name = height, rexp, 1)
+samples <- draw_samples(pop, 3, c(1, 5, 10))
+x <- create_sample_histograms(pop, samples, height, c(1, 5, 10))
+
+test_that("x axis is mapped to correct variable", {
```

Comment:
The tests should be wrapped into one function, instead of a series of different test_that

Reply:
Are you sure? 
I think it's better to have multiple test thats to give descriptive error messages for each group of tests.

Reply:
I understand what you are saying. I just have a vague idea that Tiffany said we should have one function called test_plot_sample_hist, for example, and if you run this function, all the tests in `test_that` would be run. So instead of having a bunch of `test_that`, we can just run one function and test all the things. But now I can't remember where I saw that...

Reply:
Found it in Milestone2:

"We expect each function to have a unit test function, which is named after the function being tested (e.g., if the function is named foo then the unit test function is named test_foo). The unit test function should test 2 to 3 edge cases to ensure that the function returns what is expected to the user. "

so just wrap everything in one function. Then I think everything is fine.